scripts/rollup.config.js

```diff
@@ -57,6 +59,11 @@ const options = {
     }
     defaults.module.unshift('replace');
 
+    if (NO_TEST_BUNDLE) {
+      defaults.module.unshift('strip');
```

Comment:
should we use `rollup-plugin-stub` in tests to prevent having to remove stuff from the main build? This is what we do in video.js https://github.com/videojs/video.js/blob/main/test/unit/player.test.js#L947-L955

Reply:
Looked into this, I think that we don't want to use stub anymore. The repo is archived. Also, it adds a lot of time to the build.

Reply:
Don't we want to always strip this code from the module/browser build? I think we can just remove the if statement and always add the strip plugin.

Reply:
yeah, seem reasonable. I think I wasn't sure exactly how the test/mobule/browser builds stuff worked when I first added this.